util/display/display.go

```diff
@@ -189,9 +189,8 @@ func ErrorTask() error {
 		summarizer.Error()
 		summarizer = nil
 
-		// // print the task log
-		// this is handled in a different part
```

Comment:
Originally this was commented because we were getting double log output. It was a quick fix. Later the cause of the double output was removed. But the errored task log no longer was printing correctly.